### PR TITLE
feat(movie-card): add a movie card that displays movie info

### LINKED ISSUE #7: Day 2 In Class Challenge
**Is your feature request related to a problem? Please describe.**
Create a new component called MovieCard.
The component should take the following props: title, type, posterUrl.
The component should render UI as shown below (use style or className for styling).

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Initialized state in App.js
2. Created the component MovieCard.js to display the movie info

## Purpose
To display info

Closes #007

### CODE REVIEW

**Review by connielion — COMMENTED** [review]
Looks great.

**Review by Lap343 — APPROVED** [review]
Looookin good!!!

**Review by gonzalespaulb — APPROVED** [review]
LGTM

File: movie-app/src/App.js [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
+import { getMovieByName } from "./utilities/utils";
 
 function App() {
+  const [movie, setMovie] = useState({});
 
+  useEffect(() => {
+    getMovieByName("kung pow").then((res) => setMovie(res));
+  }, []);
 
+  console.log(movie);
```
Comment (kmausolf) [inline movie-app/src/App.js:13]: Log here.

File: movie-app/src/Components/MovieCard.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+// import { getMovieByName } from "../utilities/utils";
```
Comment (kmausolf) [inline movie-app/src/Components/MovieCard.js:2]: Unused code here.

**Comment by krferrell** [comment]
should I resubmit a PR or make the changes on a future PR?

**Comment by kmausolf** [comment]
> should I resubmit a PR or make the changes on a future PR?

Let's pick this up on the next PR.  Always a good thing to scan for during PRs.

**Pull request merged**